packages/storybook/src/stories/docs/framework-integration/react-instructions.stories.mdx

```diff
@@ -42,3 +42,191 @@ $ yarn add @inovex.de/elements-react
+To configure the components of the inovex-elements we use **properties**.
+
+In this case we give our `<InoInput>` the `outline` property and its corresponding label. More properties of the `<InoInput>` can be found in our **[docs](https://elements.inovex.de/version/v7.1.0/?path=/docs/input-ino-input--playground)**.
+`<InoInput label="email" outline={true}>`.
+
+> Hint: We can customize almost all Elements components. Add icons to the `<InoInput>` element by adding a [slot and an icon](https://elements.inovex.de/version/v7.1.0/?path=/docs/input-ino-input--icons) of the `<InoIcon>` component. Play around with each components playground to see all possibilites.
+
+```jsx
+<InoInput label="password" outline>
+  <InoIcon slot="icon-trailing" icon="password"></InoIcon>
+</InoInput>
+```
+
+#### CSS Custom Properties
+
+For css customization, we can use **CSS Custom Properties** given by our `<InoInput>` component to style our outlines and labels.
+
+```scss
+InoInput {
+  --ino-input-line-color: blue;
+  --ino-input-label-color: blue;
+}
+```
+
+Most components have built in styles and themes you can choose from. Lets add a different Notification, everytime our user signs in.
+If the credentials are too short, we use the property `type="error"` of the `<InoSnackbar>` to indicate a failed login attempt. In the case of a succesfull login, we can use `type="success"`
+
+> Hint: check out our codesandbox for more insights about the functionality.
+
+```jsx
+const ValidateLogin = () => {
+  if (email.length === 0 || password.length < 8) {
+    return (
+      <InoSnackbar
+        action-text="email and/or password too short"
+        timeout={5000}
+        type="error"
+        stay-visible-on-hover="false"
+      >
+        Login failed
+      </InoSnackbar>
+    );
+  } else {
+    return (
+      <InoSnackbar
+        action-text="Welcome back!"
+        timeout={5000}
+        type="success"
+        stay-visible-on-hover="false"
+      >
+        Sucessfully logged in
+      </InoSnackbar>
+    );
+  }
+};
+```
+
+```jsx
+{
+  showSnackbar ? <ValidateLogin /> : null;
+}
+```
+
+Change the types accordingly to your wishes. The timeout property is in _ms_ and indicates when the snackbar dissapears. The timeout can be disabled by setting it to a negative value.
+More about the properties of the InoSnackbar is documented [here](https://elements.inovex.de/version/v7.1.0/?path=/docs/notification-ino-snackbar--playground).
+
+## Finishing touches
+
+### Add a logo
+
+> Hint: automatically style your logo with rounded borders by using the `rounded` property of `<InoImg>`
+
+```jsx
+<InoImg
+  alt="Elements logo"
+  decoding="async"
+  src={logo}
+  height={75}
+  width={75}
+  fallbackIcon="cloud"
+></InoImg>
+```
+
+If our image can't be loaded, it will render an `<InoIcon icon="cloud">` due to the `fallbackIcon="cloud"` property.
+The `decoding` property can either be `async`, `auto` or `sync` and sets the decode method of the native html input element.
```

Comment:
```suggestion
```

I think we can leave this part out as it might be too complicated for a beginner.